### DISCUSSION TITLE
Ability to set Valves per user

### DISCUSSION THREAD

**Discussion by hasani114** [open-webui/pipelines#230] [Ideas]
I created a valve but it seems like only admin can configure the settings for it not the user. If for example I create a pipeline that requires information that will vary per user there is no way to do this right now as far as I know. Would be nice to have permissions on valves so you can define valves for admin and valves for the user separately.

**Comment by PlebeiusGaragicus** [comment]
bump - I'd also like this.

I think we need user-facing settings for pipelines
